std_msgs/Header header

sensor_msgs/NavSatFix gps

sensor_msgs/CompressedImage image

std_msgs/UInt8 casualty_id
